libs/pageserver_api/src/models.rs

```diff
@@ -1456,4 +1532,59 @@ mod tests {
             assert_eq!(actual, expected, "example on {line}");
         }
     }
+
+    #[test]
+    fn test_aux_file_migration_path() {
```

Comment:
Not really the test I was thinking of when I left the comment on the e2e test since we this tells nothing about how the ingestion timing. But perhaps setting was never meant to be ingestion timing sensitive..?